components/ways-to-run/Sessions.vue

```diff
@@ -40,6 +40,9 @@
         </div>
       </div>
     </div>
+    <div class="c-sessions__cta">
+      <p><a class="c-button c-button--orange" href="https://help.coderdojo.com/hc/en-us/articles/360050231131-Planning-in-person-sessions-with-Covid-19-safety-measures-in-place" data-action="click" data-label="Guidance on in-person sessions" data-category="Ways to Run Downloads">Guidance on in-person sessions</a></p>
```

Comment:
does this need to be wrapped in a paragraph tag?

Reply:
I don't think it does, but I was copying prior art with other buttons down the page